src/Components/v2/index.tsx

```diff
@@ -8,6 +8,7 @@ export * from "./CountrySelect"
 export * from "./FollowIcon"
 export * from "./Footer"
 export * from "./Lightbox"
+export * from "./LoadingArea"
```

Comment:
There were quite a few places (including other apps), reaching into the artist app (which will make refactoring and moving apps around harder, later). So I just made that its own component out of the apps directory.